src/internal/remote/toBKTException.ts

```diff
@@ -42,13 +64,20 @@ export const toBKTException = async (
       return new InternalServerErrorException(
         message ?? 'Internal Server Error',
       )
+    case 502:
     case 503:
+    case 504:
       return new ServiceUnavailableException(message ?? 'Service Unavailable')
     default:
+      if (status >= 300 && status < 400) {
```

Comment:
What do you think of doing as follows?
And leave the default case only for the `UnknownException`.

```ts
case status >= 300 && status <= 400:
```

Reply:
I updated it https://github.com/bucketeer-io/javascript-client-sdk/pull/138/commits/32be4de31daf4d5e52491311e18b140db7ee041c